src/resources/views/components/fossbilling-logo.blade.php

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This causes Laravel to error out when trying to view the index page

Reply:
Ahh, really? I'm using application-logo.blade.php as reference tho, and it works just fine on my local

![image](https://user-images.githubusercontent.com/60916302/180695725-ac6cccec-7421-4a95-bca9-df1a294281b4.png)

Reply:
@BelleNottelling `php artisan key:generate` does the trick.
It works for me. However, we should document the installation process.

![image](https://user-images.githubusercontent.com/35808275/180742675-8fceb405-21a8-42e4-b098-8355131ff7ac.png)

Reply:
> @BelleNottelling `php artisan key:generate` does the trick. It works for me. However, we should document the installation process.
> 
> ![image](https://user-images.githubusercontent.com/35808275/180742675-8fceb405-21a8-42e4-b098-8355131ff7ac.png)

1. Clone / download the repo
2. run "composer install" for installing required package from composer
3. run "npm install" for installing required package from npm
4. copy and rename .env.example to .env
5. set .env file according your environment
6. run "php artisan key:generate" for generate Laravel app key
7. run "php artisan migrate:fresh" for migrate database tables

Anything I am missing?

Reply:
I assume we also not to run gulp to build the artifacts

Reply:
> 

Ah thanks, it didn't cross my mind